content/en/blog/_posts/2021-03-18-introducing-suspended-jobs.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Introducing Suspended Jobs"
+date: 2021-03-18
+slug: introducing-suspended-jobs
+url: /blog/2021/03/introducing-suspended-jobs
+---
+
+**Author:** Adhityaa Chandrasekar (Google)
+
+Jobs are a crucial part of Kubernetes API. While other controllers such as
```

Comment:
```suggestion
Jobs are a crucial part of Kubernetes API. While other kinds of workload resource, such as
```

(to cluster users the resource is what matters; the fact that it's a controller doing the work and not, eg, lots of fast-typing elves is an implementation detail :wink:)